seqr/views/utils/dataset_utils.py

```diff
@@ -321,60 +326,34 @@ def _parse_tsv_row(row):
 
 REVERSE_TISSUE_TYPE = {v: k for k, v in TISSUE_TYPE_MAP.items()}
 
-def _parse_outlier_row(row, **kwargs):
+def _parse_outlier_row(row):
     yield row['sampleID'], {mapped_key: row[key] for key, mapped_key in RNA_OUTLIER_COLUMNS.items()}
 
-def _parse_tpm_row(row, sample_id_to_tissue_type=None):
+def _parse_tpm_row(row):
```

Comment:
All RNAseq data will need a tissue-type field in the parsed output. Therefore, we remove the `kwargs` arguments for the parser functions.